src/multiselectMixin.js

```diff
@@ -560,7 +560,9 @@ export default {
 
         this.$emit('input', newValue, this.id)
       } else {
-        const optionsToAdd = group[this.groupValues].filter(not(this.isOptionDisabled || this.isSelected))
```

Comment:
The problem here was that **not** function takes **a function** as parameter and as the name suggests performs not operation on it's response and we were passing two functions separated by pipe as parameter so it always takes first function - isOptionDisabled (takes second function - isSelected only when first function is null / undefined - which is not the case here)